Icons (#1413)
elastic/docs-builder: docs/syntax/icons.md

| `i:trash:` | i:trash: |
| `i:unfold:` | i:unfold: |
| `i:unlink:` | i:unlink: |
| `i:user:` | i:user: |
| `i:users:` | i:users: |
| `i:vector:` | i:vector: |
| `i:videoPlayer:` | i:videoPlayer: |
| `i:vis_area:` | i:vis_area: |
| `i:vis_area_stacked:` | i:vis_area_stacked: |
| `i:vis_bar_horizontal:` | i:vis_bar_horizontal: |
| `i:vis_bar_horizontal_stacked:` | i:vis_bar_horizontal_stacked: |
| `i:vis_bar_vertical:` | i:vis_bar_vertical: |
| `i:vis_bar_vertical_stacked:` | i:vis_bar_vertical_stacked: |
| `i:vis_gauge:` | i:vis_gauge: |
| `i:vis_goal:` | i:vis_goal: |
| `i:vis_line:` | i:vis_line: |
| `i:vis_map_coordinate:` | i:vis_map_coordinate: |
| `i:vis_map_region:` | i:vis_map_region: |
| `i:vis_metric:` | i:vis_metric: |
| `i:vis_pie:` | i:vis_pie: |
| `i:vis_table:` | i:vis_table: |
| `i:vis_tag_cloud:` | i:vis_tag_cloud: |
| `i:vis_text:` | i:vis_text: |
| `i:vis_timelion:` | i:vis_timelion: |
| `i:vis_visual_builder:` | i:vis_visual_builder: |
| `i:warning:` | i:warning: |
| `i:warningFilled:` | i:warningFilled: |
| `i:web:` | i:web: |
| `i:wordWrap:` | i:wordWrap: |
| `i:wordWrapDisabled:` | i:wordWrapDisabled: |
| `i:wrench:` | i:wrench: |

> ## Usage Examples

Reviewer: ```suggestion
## Usage examples
```

> Icons can be used in various markdown elements.

Reviewer: ```suggestion
You can use icons in various Markdown elements.
```

### Headings

> You can include icons in headings:

Reviewer: Will they appear in the ToC?
